frontend/src/components/questionpreview/QuestionPreviewSmall.tsx

```diff
@@ -0,0 +1,104 @@
+import { QuestionDef } from "../../def/QuestionDef";
+import Section from "../section/Section";
+import Skeleton from "react-loading-skeleton";
+import { formatDate } from "../../def/converter";
+import React from "react";
+
+export default function QuestionPreviewSmall(props: { question?: QuestionDef }) {
+	return <Section className={ "glass-hover" }
+					style={ { userSelect: "none", cursor: "pointer", pointerEvents: props.question ? "all" : "none" } }
+					tabIndex={ props.question ? 0 : -1 }>
+		<div style={ { display: "flex", alignItems: "center", gap: "var(--spacing)" } }>
+			<div style={ { flex: 1 } }>
+				<p className={ "tags" }>
+					{ props.question
+						? <>
+							{ props.question.isDiscussion
+								? <span className={ "badge badge-outline" }>
+									<i className={ "fi fi-rr-comments-question" }
+									   style={ { marginRight: "calc(var(--spacing) / 2)" } }/>
+									Discussion
+							</span>
+								: <span className={ "badge badge-outline" }>
+									<i className={ "fi fi-rr-interrogation" }
+									   style={ { marginRight: "calc(var(--spacing) / 2)" } }/>
+									Question
+								</span>
+							}
+							
+							<span className={ "tags-divider" }/>
+							
+							{ props.question.tags.map((tag, index) =>
+								<span className={ "badge" } key={ index }>{ tag }</span>) }
+						</>
+						: <Skeleton width={ 150 }/>
```

Comment:
Can you decode the meaning behind these magic numbers?

Reply:
Those are proportions of how big the skeletons should be.